source/api_overview.rst

```diff
@@ -131,3 +131,16 @@ a `sensor_msgs/Imu <http://docs.ros.org/api/sensor_msgs/html/msg/Imu.html>`_
 message published at 100Hz.
 
 The IMUs are not present in the simulated robot.
+
+Resetting Breakers
+-------------
+
+There are 3 breakers governing power on the fetch. One each for the arm, gripper 
+and base. If the motors are commanded to perform beyond their limits they will 
+shut down for safety. To reset them you will either need to toggle the e-stop or 
+use the following service calls:
+
+
+ * $ rosservice call /arm_breaker false && rosservice call /arm_breaker true
```

Comment:
Actually, disabling the breaker is the only way to clear the error flag so that it can be re-enabled